### PR TITLE
This contains my game file, MyGame.py

### PR DESCRIPTION
**Pull request by k0789885**

### CODE REVIEW

File: k0789885/MyGame.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+print "1. Warrior"
+print "2. Archer"
+print "3. Mage"
+
+
+
+def bar():
+    print "You push the doors open and are greeted to a room of drunkards and maidens."
+    print "There is a man in a dark cloak in the corner and the bartender is at the counter."
+    while choice != 1 and choice != 2 and choice != 3:
```
Comment (chrisortman) [inline k0789885/MyGame.py:14]: consider:

`choice not in [1,2,3]`

Does one seem easier to understand than the other?

**Review by chrisortman — APPROVED** [review]
Looks like a good start, classes will probably make your life a lot easier with some of this.

Watch out for naming your variables with uppercase letters. It is going to cause you confusion soon because we will be defining our own types

**Comment by k0789885** [comment]
I attempted the choice not in [1,2,3,4] and it sticks me in a loop that I
can get out of but it automatically puts me back in the loop.

**Comment by k0789885** [comment]
Hmm?

**Pull request merged**